src/confetti.js

```diff
@@ -70,13 +71,13 @@
     var val = String(str).replace(/[^0-9a-f]/gi, '');
 
     if (val.length < 6) {
-        val = val[0]+val[0]+val[1]+val[1]+val[2]+val[2];
+      val = val[0] + val[0] + val[1] + val[1] + val[2] + val[2];
```

Comment:
Please don't change any random spacing. This adds unnecessary noise and hides the actual code being added.